common/include/PS2Edefs.h

```diff
@@ -181,10 +181,12 @@ void CALLBACK GSsetGameCRC(int crc, int gameoptions);
 // controls frame skipping in the GS, if this routine isn't present, frame skipping won't be done
 void CALLBACK GSsetFrameSkip(int frameskip);
 
-// if start is 1, starts recording spu2 data, else stops
+// Starts recording GS frame data
 // returns a non zero value if successful
-// for now, pData is not used
-int CALLBACK GSsetupRecording(int start, void *pData);
+int CALLBACK GSsetupRecording(std::wstring& filename);
```

Comment:
I'm concerned about the usage of wstring as outlined here:
https://stackoverflow.com/a/50407375

This goes for your other usages of wstring as well. It's also inconsistent with the current API used by the OSD.